nirc_ehr/resources/queries/study/prcSchedule.sql

```diff
@@ -0,0 +1,13 @@
+
+SELECT
+    po.procedureRecord,
+    po.Id,
+    po.procedure,
+    po.windowStart,
+    po.windowEnd,
+    po.orderedby,
+    po.remark,
+    po.objectid,
+    CASE WHEN po.qcstate.label = 'Completed' THEN 'Completed' ELSE '' END as status
+FROM prc_order po
+WHERE now() >= windowStart AND now() <= windowEnd
```

Comment:
Currently, 'Today's Procedure Schedule' is showing Procedures that are completed, is that expected?

Reply:
yes that's how it's meant to be until user testing says it should be otherwise